kubernetes-templates/ghost/0/rancher-compose.yml

```diff
@@ -0,0 +1,29 @@
+.catalog:
+  name: Ghost
+  version: 0.7.8-rancher1
+  description: Open source publishing platform
+  questions:
+    - variable: "mysql_root_pass"
+      label: "MySQL root password"
+      required: true
+      type: "string"
+    - variable: "ghost_db"
+      label: "Ghost database name on mysql"
+      required: true
+      type: "string"
+      default: "ghost"
+    - variable: "ghost_db_user"
+      label: "Ghost database username"
+      required: true
+      type: "string"
+      default: "ghostuser"
+    - variable: "ghost_db_pass"
+      label: "Ghost database password"
+      required: true
+      type: "string"
```

Comment:
It's up to you if you want to specify a default password. If you do (and make it password type) then it will be masked, so add the default to the description.

We usually add descriptions to each variable, but it is not strictly required.